scripts/build-upgrade-image.sh

```diff
@@ -93,6 +106,12 @@ cp version.info.template version.info
 VERSION=$(dpkg -f "$(find debs/ -name 'delphix-entire-*' | head -n 1)" version)
 sed -i "s/@@VERSION@@/$VERSION/" version.info
 
+cp verification-version.info.template verification-version.info
```

Comment:
I think it'd be good to add a comment to explain why we're not using `version.info` for this info.